definitions/output/all/parsed_css.js

```diff
@@ -0,0 +1,22 @@
+publish("parsed_css", {
+    type: "incremental",
+    protected: true,
+    schema: "all",
+    bigquery: {
+        partitionBy: "date",
+        requirePartitionFilter: true
```

Comment:
I added some extra clustering here:

```suggestion
        clusterBy: ["client", "is_root_page", "rank", "page"],
        requirePartitionFilter: true
```